source/features/show-user-top-repositories.tsx

```diff
@@ -5,20 +5,18 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {getCleanPathname} from '../github-helpers';
 
-function buildUrl(queryField: string): URL {
-	const url = new URL('/search', location.href);
-	url.searchParams.set('o', 'desc');
-	url.searchParams.set('q', `user:${getCleanPathname()}`);
-	url.searchParams.set('s', queryField);
-	url.searchParams.set('type', 'Repositories');
+function buildUrl(): URL {
+	const url = new URL('/' + getCleanPathname(), location.href);
```

Comment:
I think I may like the search page, since searching for a language and other filters are simpler.

WDYT?

Reply:
I have no opinion about this. @fregante?

Reply:
Since we’re on the profile page, we should stay on the profile page